package.json

```diff
@@ -39,7 +39,6 @@
     "angular-ui-sortable": "~0.16.1",
     "angular.validators": "~4.4.3",
     "array-includes": "~3.0.3",
-    "base64-js": "~1.2.3",
```

Comment:
you can also remove `text-encoder-lite` (only used by `http_api/compat.js`, removed here)